doc/digital-receipt/implementation/digital-receipt-implementation.md

```diff
@@ -0,0 +1,667 @@
+
+<details>
+<summary>Response details (shortened):</summary> 
+  
+```
+"{
+"ftCashBoxID":"124358e8-9cbd-4332-9076-7ed8b72306ac"
+"ftQueueID":"c719b59a-9ff0-891a-98a1-030776d8c46f"
+"ftQueueItemID":"f838d7c8-3113-4c3a-914a-46d7de555d11"
+"ftQueueRow":"41"
+"cbTerminalID":"T2"
+"cbReceiptReference":"pos-action-identification-02"
+"ftCashBoxIdentification":"bonstart"
+"ftReceiptIdentification":"ft27#IT11"
+"ftReceiptMoment":"2023-08-03T13:12:56.3989064Z"
+}"
+```
+</details>
+
+## Configure POS API Helper 
+
+The POS API Helper is available in all countries. This Helper is responsible for uploading data from the local Queue to the digital receipt endpoint. This Helper is configured in the fiskaltrust.Portal and assigned to each CashBox that uses digital receipts. The POS API Helper changes to an direct upload behavior of the digital receipts.
+
+To proceed with the configuration, login to your fiskaltrust.Portal account first. 
+
+### Queue 
+
+| Step  | Description |
+| ------------- | ------------- |
+| 1  | Navigate to the configuration section and go to Queue  |
+| 2  | Configure Queue  |
+| 3  | Copy the URLs to your local machine (Required for CashBox configuration later)   |
+| 4  | For all countries: Change port to the next free port (+1) and <br/> a.	if no suffix exists after the port: add the suffix "/name_queue" to the URL ("name" can be freely chosen) <br/> b.	if suffix already exists: add the suffix "_queue" to the URL  |
+| 5  | Germany & France only: Change grpc port to the next free port (if port is free no need to go up to the next free port) and add the suffix "/name_queue" to the URL ("name" can be freely chosen)  |
+| 6  | Save changes  |
+
+### Helper 
+
+| Step  | Description |
+| ------------- | ------------- |
+| 1  | Navigate to Helper  |
+| 2  | Create new helper  |
+| 3  | Add description  |
+| 4  | Select  package name "fiskaltrust.service.helper.posapi"   |
+| 5  | Select latest package version   |
+| 6  | Select the outlet of CashBox    |
+| 7  | Save configuration   |
+| 8  | Klick configure helper   |
+| 9  | All Counties: Insert the previously saved Queue URLs to the Helper URLs and add the suffix "/name" to the URL (analogue to the naming in queue configuration). Germany & France only: Add also GRPC URL with next free port and add the suffix "/name" to the URL (analogue to the naming in queue configuration).   |
+| 10  | Save configuration and close   |
+
+### CashBox 
+
+| Step  | Description |
+| ------------- | ------------- |
+| 1  | Navigate to CashBox   |
+| 2  | Select your CashBox and click edit by list  |
+| 3  | Navigate to Helpers  |
+| 4  | Activate the POS API Helper  |
+| 5  | Save configuration  |
+| 6  | Klick rebuild configuration  |
+
+### Restart
+
+Restart the fiskaltrust.Middleware to apply the changes. 
+
+# Create receipts with the print endpoint – receive receipt with response via POS API (preferred) 
```

Comment:
Since this is a "subtitle", it should be `##` i think (and the ones below should also have one more #)